src/styles/main.scss

```diff
@@ -0,0 +1,78 @@
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+$calendar-day-width: 100px;
+
+.calendar {
+  width: $calendar-day-width * 7 + 6px;
```

Comment:
The width calculation for the `.calendar` class includes `6px` added to `$calendar-day-width * 7`. Ensure that this additional 6px is intentional and aligns with your design requirements, as it might affect the overall layout.